changelog.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 - [[#3885] Allow Adding a Single Service Level Interceptor](https://github.com/ballerina-platform/ballerina-standard-library/issues/3885)
+- [[#4206] Improve GraphQL Document Validation Logics](https://github.com/ballerina-platform/ballerina-standard-library/issues/4206)
```

Comment:
Let's make this specific

Reply:
`Skip Validation for Unused Operations in the Document`?

Reply:
Maybe `Skip Additional Validation for Unused Operations in the Document`? Because we do some validations for the unused operations?